inc/default-styles.php

```diff
@@ -220,102 +160,109 @@ static function row_style_fields( $fields ) {
 			'name'        => __( 'Border Color', 'siteorigin-panels' ),
 			'type'        => 'color',
 			'group'       => 'design',
-			'description' => __( 'Border color of the row.', 'siteorigin-panels' ),
+			'description' => sprintf( __( 'Border color of the %s.', 'siteorigin-panels' ), strtolower( $label ) ),
 			'priority'    => 10,
 		);
 
 		return $fields;
 	}
 
-	static function cell_style_fields( $fields ) {
-		$fields['class'] = array(
+	static function row_style_fields( $fields ) {
+		// Add the general fields
+		$fields = wp_parse_args( $fields, self::get_general_style_fields( 'row', __( 'Row', 'siteorigin-panels' ) ) );
+
+		//Should we remove this? Or move it's existing value to new Cell Styles?
```

Comment:
Can remove this comment. We ended up keeping this as applying to all cells, but it will be overridden by the specific cell styles `class` field.